examples/nlp/language_modeling/conf/megatron_gpt_inference.yaml

```diff
@@ -38,6 +38,55 @@ web_port: 9889 # the port number of the web server
 chat: False # use the chat interface
 chatbot_config:
   value: False   # whether to inject the value attributes
+  attributes:
```

Comment:
is it possible to move these attributes to some other file?
don't want to end up bloating the main gpt_inference config file by adding multiple value attributes (esp if we end up adding more)

Reply:
It is meant to be an example and not a full list. I can remove a few items. The user can always use a customized yaml file.

Reply:
okay, outside this PR, IMO we should handle the chatbot configs in a new file.... bloating this one up for chatbot does not feel good...